README.md

```diff
@@ -97,6 +97,43 @@ Other available operations may be discovered by running:
 ./project.rb
 ```
 
+#### API: Faster API startup for MacOS
+The above steps for starting the API server can take upwards of 8-10 minutes on MacOS, most likely due to performance issues with Docker for Mac. Follow these steps to set up your developer environment to start the API server outside of docker. A full restart should take ~30 seconds with this method.
+
+All commands should be run from `workbench/api`
+
+##### Setup
+* Install Java 8
+* Add following to `~/.bash_profile`. Note: Your Java8 library directory may be different
+    * ```Shell
+      export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_201.jdk/Contents/Home
+      export OVERWRITE_WORKBENCH_DB_HOST=true
+      ```
+    * Source your bash profile or open a new terminal
+      ```Shell
+      source `~/.bash_profile`
+      ```
+* Install Java App Engine components
+    * ```Shell
+      gcloud components install app-engine-java
+      ```
+* Generate Google Cloud access token
+    * ```Shell
+      ./project.rb get-test-service-creds
+      ```
+
+##### Usage
+* If you have schema migrations or pending configuration updates, run through the [normal docker developer startup process](#api-dev-appengine-appserver) at least once successfully. This is typically only necessary when switching between branches.
```

Comment:
is normal docker developer startup process "./project.rb dev-up" in /api? is there an easier command to run migrations without also starting the api, so i can then start it through this new process?

Reply:
there is definitely something quicker we can run but I haven't worked out what those commands are yet